src/components/DocumentList/DocumentListScreen.tsx

```diff
@@ -19,10 +20,17 @@ export const DocumentListScreen: FunctionComponent<DocumentListScreen> = ({
 }) => {
   return (
     <ScreenView>
-      {documentItems.length > 0 ? (
-        <DocumentList documents={documentItems} navigateToDoc={navigateToDoc} />
+      {documentItems ? (
+        documentItems.length > 0 ? (
```

Comment:
A bit hard to read here, wondering if we can don't nest it. Instead can be another class `<ResolvedDocumentList />` to merge the empty and the populated document list